admin-perspective/4.4/notification-drawer/notification-drawer.md

```diff
@@ -0,0 +1,28 @@
+# Notification Drawer
+
+## Masthead Icon
+![](img/notifications-closed.png)
+- When alerts are firing, a special signifier icon appears in the masthead. Clicking on the icon opens the notification drawer.
+
+## Critical Alerts
+![](img/notifications-open.png)
+- As this drawer is an endopoint for alertmanager, silenced and pending alerts do not appear here.
+- Clicking on an alert takes the user to the details page for the alert.
+- The alertmanager config could potentially be set up to send non-critical alerts here, but by default that would not be the case.
```

Comment:
Why not show all alerts?

Reply:
The urgency of warning alerts is ideally on the scale of several days, Ideally we would encourage people to set up an alert receiver to get these into some kind of ticketing system. I worry that including warnings in this drawer would cause alert fatigue and people would start to ignore them.

Reply:
I don't think that's the case - for instance, NodeDown is a warning.  You should take action on warnings pretty fast.  We have silences for a reason.  If your nodes are going down, you have a problem.

Reply:
@sichvoge what are your thoughts on how much we should surface here? I guess ideally, the user would have some sort of mobile phone alert set up for things that are _really_ critical and  could look to the UI for a list of everything.